fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/FileCardTokens.kt

```diff
@@ -0,0 +1,144 @@
+package com.microsoft.fluentui.theme.token.controlTokens
+
+import android.os.Parcelable
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.microsoft.fluentui.theme.FluentTheme
+import com.microsoft.fluentui.theme.token.FluentAliasTokens
+import com.microsoft.fluentui.theme.token.FluentGlobalTokens
+import kotlinx.parcelize.Parcelize
+
+data class FileCardInfo(
+    val isPreviewAvailable: Boolean = true,
+    val cardType: CardType = CardType.Elevated
+)
+
+@Parcelize
+open class FileCardTokens : BasicCardTokens(), Parcelable {
```

Comment:
FileCardTokens should extend IControlToken.

Reply:
done